### PR TITLE
Fix cancel button and use VaadinIcons instead of FontAwesome

### PR DESCRIPTION
**Pull request by mstahv**

### CODE REVIEW

File: complete/src/main/java/hello/CustomerEditor.java [hunk 1]
```diff
@@ -1,16 +1,17 @@
 package hello;
 
+import org.springframework.beans.factory.annotation.Autowired;
```
Comment (snicoll) [inline complete/src/main/java/hello/CustomerEditor.java]: That import seems unnecessary. I'll remove it

File: complete/src/main/java/hello/CustomerEditor.java [hunk 2]
```diff
@@ -78,7 +80,7 @@ public final void editCustomer(Customer c) {
 		final boolean persisted = c.getId() != null;
 		if (persisted) {
 			// Find fresh entity for editing
-			customer = repository.findOne(Example.of(c)).get();;
+			customer = repository.findById(c.getId()).get();
```
Comment (snicoll) [inline complete/src/main/java/hello/CustomerEditor.java]: I am confused. Why was this working before?

File: complete/src/main/java/hello/CustomerEditor.java (hunk 1 above)
Comment (mstahv) [reply complete/src/main/java/hello/CustomerEditor.java]: Oops, I didn't fetch your cleanup change.

File: complete/src/main/java/hello/CustomerEditor.java (hunk 2 above)
Comment (mstahv) [reply complete/src/main/java/hello/CustomerEditor.java]: I broke the cancel button when I did the Spring 5 repository changes. Cancel button just fetches the previous state of the entity from the db, and the example query checks for all the properties, not just id.

Comment (snicoll) [reply complete/src/main/java/hello/CustomerEditor.java]: Got it. I think I haven't tried that when I ran the thing locally.

**Pull request closed without merging**